docs/FAQ.md

```diff
@@ -90,6 +90,24 @@ See the below GIF to better sync with above steps.
 
 ![selected_footer_autocomplete](https://user-images.githubusercontent.com/55916430/116669526-53cfb880-a9bc-11eb-8073-11b220e6f15a.gif)
 
+### List of designated prefixes currently supported in ZT
+
+Now that you have a basic understanding of how autocompletes in ZT work, let's explore what all sorts of autocomplete prefixes ZT support, as of now. Note that the kinds of prefix supported also differ within each compose box. Thus, the prefixes listed below are grouped by type of compose box.
+
+You could refer to the above section to try out each of the below prefixes!
```

Comment:
This style of text is probably more appropriate in a tutorial.

Probably all we need is text somewhere which indicates that:
* in the message body (I don't think that word came up, but it seems OK?) we use prefixes
* for the recipient boxes (1, 2, 3) the prefixes are assumed

Note that with recipient boxes there is no "prefix" (as above bullets) so perhaps this title could be "Areas with autocomplete support" ?

Reply:
Sounds good. I was just framing the two bullets in a similar way. I wanted to query if recipient boxes sound internal? Does "recipient header" make sense?